packages/typespec-vscode/package.json

```diff
@@ -261,5 +277,8 @@
     "vitest": "^3.0.5",
     "vscode-languageclient": "~9.0.1",
     "yaml": "~2.7.0"
+  },
+  "dependencies": {
```

Comment:
devDependency doesnt work?

Reply:
I think it works. I chose `dependencies`, because its meaning is for packages required for your application to run in production. `devDependencies` is for packages needed only for local development and testing.

Reply:
changed to `devDependencies` per discussion